api/src/emails/api.py

```diff
@@ -22,7 +22,9 @@ def send_email():
         return jsonify(err.messages), 422
 
     msg = Message(valid_email_request['subject'], recipients=valid_email_request['recipients'],
-                                                   sender=valid_email_request['managerEmail'])
+                                                   sender=valid_email_request['managerEmail'],
+#                                                    reply_to='qiang_wang@taylor.edu')
```

Comment:
Should this comment remain in the code?

Reply:
This should be removed, which is for testing.